pkgs/development/python-modules/plyara/default.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  wheel,
+  ply,
+}:
+
+buildPythonPackage rec {
+  pname = "plyara";
+  version = "2.1.1";
+  pyproject = true;
+
```

```suggestion

  disabled = pythonOlder "3.10";

```
A nice to have for those still using older Python versions